docs/extend/extend-cli/cli-devTutorials.md

```diff
@@ -8,13 +8,27 @@ You can contribute to Zowe CLI in the following ways:
 - Develop a plug-in that users can install to Zowe CLI.
```

Comment:
Sounds odd. How about "Develop a plug-in to contribute to Zowe CLI"?

Reply:
It does sound weird.

Maybe it could be simplified to: "Develop a Zowe CLI plug-in".